spec/controller/search_controller_spec.rb

```diff
@@ -3,12 +3,12 @@
 
 describe SearchController, :type => :controller do
   def stub_client
-    stub_search_client = stub("search", search: { "results" => [] })
+    stub_search_client = stub("search", unified_search: { "results" => [] })
     controller.stubs(:search_client).returns(stub_search_client)
   end
 
   def do_search(term = 'search-term')
-    get :index, prefix: 'service-manual', q: term
+    get :index, prefix: 'service-manual', q: term, filter_manual: "service-manual"
```

Comment:
Is this correct?  From reading the code, I don't think we expect a `filter_manual` query param to be passed to the controller.